noop/noop_url.go

```diff
@@ -0,0 +1,18 @@
+package noop
+
+import (
+	"net/url"
+
+	"github.com/uw-labs/substrate"
+	"github.com/uw-labs/substrate/suburl"
+)
+
+func init() {
+	suburl.RegisterSink("noop", newNoopSink)
+}
+
+func newNoopSink(u *url.URL) (substrate.AsyncMessageSink, error) {
+	return noopSinker()
+}
+
+var noopSinker = NewAsyncMessageSink
```

Comment:
To be consistent, I've added this variable as in the other files. But I am not sure if there is a specific reason around it.

Reply:
It's to aid testing, but since the noop sink doesn't take any options, you don't need it here...